src/e2e/java/teammates/e2e/cases/StudentProfilePageE2ETest.java

```diff
@@ -40,7 +40,11 @@ public void testAll() {
         profilePage.waitForUploadEditModalVisible();
 
         profilePage.editProfilePhoto();
-        profilePage.verifyPhotoSize("220px", "220px");
+        profilePage.verifyPhotoSize(220, 220);
```

Comment:
This block can also reuse the `uploadAndVerify` function. `editProfilePhoto` is a separate test operation that can be done after checking the photo size.